### ISSUE TITLE
Port to Fastai 2

### ISSUE THREAD

**Issue by tgalery** [n-waves/multifit#82]
Hi I was wondering if there's an attempt to update this codebase to fastai 2. I'm trying to run that on updated libraries but it seems trickier than I anticipated.

**Comment by Xalsar** [comment]
Yes @tgalery I do not think you will be able to use multifit in fastai 2. By now we will have to stick to the classical ULMFIT way.

[Jeremy Howard explains it very well here.](https://www.youtube.com/watch?v=MDX_x6rKXAs&list=PLtmWHNX-gukKocXQOkQjuVxglSDYWsSh9&index=11)

Do you have any other alternative?
